TodoApi/TodoApi.csproj

```diff
@@ -11,6 +13,7 @@
     <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="7.0.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="7.0.0" />
+    <PackageReference Include="Microsoft.NET.Build.Containers" Version="0.2.7" />
```

Comment:
This package is currently required to publish an app as a container according to this [doc](https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container).